R/endpoints.R

```diff
@@ -32,12 +32,16 @@ model_options <- function(input) {
 calibration_options <- function(iso3) {
   tryCatch({
     json_verbatim(
-      naomi.options::get_controls_json("calibration", iso3, NULL, NULL))
+      get_controls_json("calibration", iso3, NULL, NULL))
   }, error = function(e) {
     hintr_error(e$message, "INVALID_CALIBRATION_OPTIONS")
   })
 }
 
+get_controls_json <- function(...) {
```

Comment:
What's the idea with this wrapper? You can also achieve this by doing

```
#' @importFrom naomi.options get_controls_json
```

if you just want to drop the `::` for some reason

Reply:
This is just because with mockr you cannot mock functions from other packages. As discussed let's go with this for now and can move to using stub and avoid `with_mock` in the future.